crates/sui-core/src/authority.rs

```diff
@@ -1578,6 +1598,7 @@ impl AuthorityState {
         let state = AuthorityState::new(
```

Comment:
Slightly unrelated to this PR, but is the comment above accurate anymore?

Reply:
definitely not!